repose-aggregator/tests/release-verification/src/config/scripting.cfg.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<scripting xmlns="http://docs.openrepose.org/repose/scripting/v1.0"
+           language="groovy">
+    if (request.getHeader('X-Auth-Token') == null) {
+        request.addHeader('X-Auth-Token', '12345678901234567890')
+    }
+</scripting>
```

Comment:
Wow. If the Scripting filter does not call `filterChain.doFilter()` and does not reject the request itself, then Repose simply hands it off to the origin service for processing. So as it is currently the request went through the Scripting filter and the Origin service, but bypassed the Keystone and API-Validator filters.